<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="AlphabetIcon">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>6</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Icons/Alphabet.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>14</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Add Custom Fonts</string>
   </property>
  </widget>
  <widget class="QListWidget" name="CustomList">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>70</y>
     <width>501</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(86, 86, 86);</string>
   </property>
   <property name="text">
    <string>Added Font</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_AddList">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>233</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>➕</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_RemoveList">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>233</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>➖</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_FontListDone">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>233</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Icons/check.png</normaloff>../Icons/check.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
